content/en/about/supported-releases/index.md

```diff
@@ -53,15 +53,16 @@ current `<minor>` release. A patch is usually a small change relative to the `<m
 | master          | No, development only  |                     |                     |                               |                                               |
 | 1.9             | N/A                   | ~Feb 2021(Expected) | ~Aug 2021(Expected) | 1.17, 1.18, 1.19, 1.20        | 1.15, 1.16                                    |
 | 1.8             | Yes                   | November 10, 2020   | ~May 2021(Expected) | 1.16, 1.17, 1.18, 1.19        | 1.15                                          |
-| 1.7             | Yes                   | August 21, 2020     | ~Feb 2021(Expected) | 1.16, 1.17, 1.18              | 1.15                                          |
+| 1.7             | Yes                   | August 21, 2020     | Feb 19, 2021         | 1.16, 1.17, 1.18              | 1.15                                          |
 | 1.6             | No                    | May 21, 2020        | November 23, 2020   | 1.15, 1.16, 1.17, 1.18        |                                               |
 | 1.5 and earlier | No                    |                     |                     |                               |                                               |
 
 ## Releases without known Common Vulnerabilities and Exposures (CVEs)
 
 | LTS Release                | Patched versions with no known CVEs  |
```

Comment:
I recall some discussion recently about removing LTS from this and or other pages. We seem to have at least removed that from the vocabulary, but still seem to call a 1.x either a minor (on this page) or major (on https://preliminary.istio.io/latest/news/). We should probably decide on which it should be (maybe this was already decided, just not implemented?) and change everything to match.

There is also a mention of LTS at the end of an earlier paragraph.

Reply:
Removed reference to LTS. I know @therealmitchconnors is suggesting LTS in the future, but for now the Istio community doesn't have such a concept. Only a supported window. This was in a previous PR. I've changed such references to "minor" releases.